packages/app/src/vis-packs/core/hooks.ts

```diff
@@ -160,3 +160,19 @@ export function useSlicedDimsAndMapping(
     [dimMapping, dims]
   );
 }
+
+export function useFillValue(dataset: Dataset): number | undefined {
+  const { attrValuesStore } = useDataContext();
+  const fillValueArray = attrValuesStore.getSingle(dataset, '_FillValue');
+
+  if (!Array.isArray(fillValueArray)) {
+    return undefined;
+  }
+
+  const fillValue = fillValueArray[0];
+  if (typeof fillValue !== 'number') {
+    return undefined;
+  }
+
+  return fillValue;
+}
```

Comment:
This hook may have to be changed. In the file supplied in #1381, `_FillValue` is a array of one value but [the specs](https://docs.unidata.ucar.edu/netcdf-c/current/attribute_conventions.html) tell that:

> If _FillValue is defined then it should be scalar and of the same type as the variable.

So this array parsing may not be needed

Reply:
Then it's probably worth also supporting `_FillValue` being a `number`.

Reply:
Yeah, might as well support both... Maybe it was allowed by an old version of the spec.